src/internal/detect.rs

```diff
@@ -23,16 +26,28 @@ pub fn detect() {
 
     // If pacnew files are found, warn the user and prompt to pacdiff
     if !pacnew.is_empty() {
-        let choice = prompt!(default false, "It appears that at least one program you have installed / upgraded has installed a .pacnew/.pacsave config file. Would you like to run pacdiff to deal with this? You can always deal with this later by running `sudo pacdiff`");
+        sp.stop_bold("It appears that at least one program you have installed / upgraded has installed a .pacnew/.pacsave config file. These are created when you have modified a program's configuration, and a package upgrade could not automatically merge the new file.");
+
+        let choice = prompt!(default false, "Would you like to run pacdiff to deal with this? You can always deal with this later by running `sudo pacdiff`");
         if choice {
-            warn!("Unless you've set an alternative using the DIFFPROG environment variable, pacdiff uses `vimdiff` by default to edit files for merging. Make sure you know how to exit vim before proceeding");
-            let cont = prompt!(default false, "Continue?");
-            if cont {
+            if env::var("PACDIFF_WARNING").unwrap_or_else(|_| "1".to_string()) != "0" {
+                warn!("Pacdiff uses `vimdiff` by default to edit files for merging. Make sure you know how to exit vim before proceeding!");
```

Comment:
Shouldn't the prompt just tell them how to close it? It feels kinda counterproductive to tell the user that they should know how to quit it but then don't give any source how to quit it

Reply:
How about changing the line to this:
"Pacdiff uses `vimdiff` by default to edit files for merging. To exit vimdiff, press the following key combination: ESC, :qa!, ENTER"

Reply:
Yeah that sounds good